packages/react-core/src/components/Form/Form.tsx

```diff
@@ -9,24 +9,34 @@ export interface FormProps extends React.HTMLProps<HTMLFormElement> {
   className?: string;
   /** Sets the Form to horizontal. */
   isHorizontal?: boolean;
-  /** Flag to limit the max-width to 500px. */
+  /** Limits the max-width of the form. */
   isWidthLimited?: boolean;
+  /** Sets a custom max-width when using isWidthLimited. */
+  maxWidth?: string;
```

Comment:
We can update this prop to be a union later on with default, sm, md, lg, or with an object with max-widths at different breakpoints. That would not be breaking.